Duckling/Time/PL/Rules.hs

```diff
@@ -761,7 +761,7 @@ ruleDaybeforeyesterdaySingleword = Rule
   , pattern =
     [ regex "przedwczoraj"
     ]
-  , prod = \_ -> tt $ cycleNth TG.Day 2
+  , prod = \_ -> tt . cycleNth TG.Day $ - 2
```

Comment:
why did you change this from 2 to (-2)?

also, can you change the `.` back to `$`? It fails our internal linter.

Reply:
`przedwczoraj` means "two days ago" so there was a bug. There wasn't test for that so I wrote it and now it is working as expected.

Reply:
I also changed `$` with `.` so it is ready to review @chessai